app.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+var app = document.getElementById('app');
+var clicksRemaining = 25;
+var secondToLastPhotos = [];
+var previousPhotos = [];
+var displayedPhotos = [];
+
+var photos = [
+  new Photo('bag head', 'bag.jpg'),
+  new Photo('banana chopper', 'banana.jpg'),
+  new Photo('bathroom party', 'bathroom.jpg'),
+  new Photo('rain boots', 'boots.jpg'),
+  new Photo('breakfast', 'breakfast.jpg'),
+
+  new Photo('bubblegum', 'bubblegum.jpg'),
+  new Photo('chair', 'chair.jpg'),
+  new Photo('cthulhu', 'cthulhu.jpg'),
+  new Photo('dog-duck', 'dog-duck.jpg'),
+  new Photo('dragon', 'dragon.jpg'),
+
+  new Photo('pen', 'pen.jpg'),
+  new Photo('pet-sweep', 'pet-sweep.jpg'),
+  new Photo('scissors', 'scissors.jpg'),
+  new Photo('shark in the dark', 'shark.jpg'),
+  new Photo('tauntaun', 'tauntaun.jpg'),
+
+  new Photo('sweep', 'sweep.png'),
+  new Photo('unicorn', 'unicorn.jpg'),
+  new Photo('usb', 'usb.gif'),
+  new Photo('water-can', 'water-can.jpg'),
+  new Photo('wine-glass', 'wine-glass.jpg'),
+];
+
+
+renderPhotoChoices();
+
+// make a constructor for photo
+function Photo(name, filename){
+  this.name = name;
+  this.src = './imgs/' + filename;
+  this.clickCount = 0;
+  this.displayCount = 0;
+}
+
+
+
+function getRandomIndex(list) {
+  return Math.floor(Math.random() * list.length);
+}
+
+// get three random photos
+function generateRandomPhotos(){
+  photos = photos.concat(secondToLastPhotos);
+  secondToLastPhotos = previousPhotos;
+  previousPhotos = displayedPhotos;
+  displayedPhotos = [];
+
+  // create a var nextPhoto to keep track of the next Photo we take out of photos
+  // splice out an photo object (wich removes it from photos)
+  var nextPhoto = photos.splice(getRandomIndex(photos), 1);
+  // concat the array returned by splice onto photos onScreen
+  displayedPhotos = displayedPhotos.concat(nextPhoto);
+  nextPhoto = photos.splice(getRandomIndex(photos), 1);
+  displayedPhotos = displayedPhotos.concat(nextPhoto);
+  nextPhoto = photos.splice(getRandomIndex(photos), 1);
+  displayedPhotos = displayedPhotos.concat(nextPhoto);
+}
+
+
+function handlePhotoClick(event){
+  var image = event.target;
+  var displayedPhotosIndex = image.getAttribute('photos-on-screen-index');
+  displayedPhotos[displayedPhotosIndex].clickCount++;
+
+  clicksRemaining--;
+  if (clicksRemaining > 0){
+    renderPhotoChoices();
+  } else {
+    app.textContent = '';
+    clearAndDisplay();
+  }
+}
+
+function renderPhotoChoices(){
+
+  generateRandomPhotos();
+
+
+  // re populate the app div
+  var imageElement;
+  for(var i=0; i < displayedPhotos.length; i++){
+
+    displayedPhotos[i].displayCount++;
+
+    // img element
+    imageElement = document.createElement('img');
```

Comment:
Instead of creating a new element, get your existing img element and update the src property.  You would only need to change this line, remove the event listener/append child lines.  This would be a more efficient method of changing images than the alternative fix.

If you do not update the src of an existing element, you would need to remove the currently displayed images.  This is a slower process though as it must clear then re-make all your images instead of simply update the src.